json-to-messages/javascript/src/JSONTraverse.ts

```diff
@@ -8,15 +8,16 @@ import { traverseFeature as traverseBehaveFeature } from './behave/JSONTraverse'
 import IPredictableSupportCode from './IPredictableSupportCode'
 import IAstMaker from './IAstMaker'
 import { IdGenerator, messages } from '@cucumber/messages'
+import { Platform } from './types'
 
 export default function traverseFeature(
-  sourceLang: string,
+  platform: Platform,
```

Comment:
Platform typically means Windows/Unix/OSx. Would implementation be a better word?

Reply:
Or dialect?

Reply:
Yes, I think `implementation` is better. Let's change that too before merging this (adding a TODO above)